doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-debugging-tips.rst

```diff
@@ -3,71 +3,79 @@
+or ssh into the teuthology server::
 
     ssh <username>@teuthology.front.sepia.ceph.com
 
-and accessing `teuthology archives`_, for example::
-  
-  nano /a/teuthology-2021-01-06_07:01:02-rados-master-distro-basic-smithi/
+and access `teuthology archives`_, like this for example:
 
-.. note:: This would require Sepia lab access. To know how to request it, see:
+  .. prompt:: bash $
+
+     nano /a/teuthology-2021-01-06_07:01:02-rados-master-distro-basic-smithi/
+
+.. note:: This requires you to have access to the Sepia lab. To learn how to
+          request access to the Speia lab, see: 
           https://ceph.github.io/sepia/adding_users/
 
-On pulpito, the jobs in red specify either a failed or dead job.
-Here, a job is combination of daemons and configurations that are formed using
+On pulpito, the jobs in red specify either a failed job or a dead job.
+A job is combination of daemons and configurations that are formed using
 `qa/suites`_ yaml fragments.
-Taking these configurations, teuthology runs the tasks that are present in
+Teuthology uses these configurations and runs the tasks that are present in
 `qa/tasks`_, which are commands used for setting up the test environment and
 testing Ceph's components.
-These tasks help us in covering large subset of usecase scenarios and hence
-exposing the bugs which were uncaught by `make check`_ testing.
+These tasks cover a large subset of usecase scenarios and help to  
+expose the bugs that were not caught by `make check`_ testing.
 
 .. _make check: ../tests-integration-testing-teuthology-intro/#make-check
 
-A job failure hence might be because of:
+A job failure might be caused by one or more of the following:
 
 * environment setup(`testing on varied systems<https://github.com/ceph/ceph/tree/master/qa/distros/supported>_`):
   testing compatibility with stable realeases for supported versions.
 
 * permutation of config values: for instance, qa/suites/rados/thrash ensures to
-  test Ceph under stressful workload, so that we be able to catch corner case
+  test Ceph under stressful workload, so that we are able to catch corner-case
   bugs.
-  The final setup config yaml that would be used for testing can be accessed
-  at::
+  The final setup config yaml used for testing can be accessed at::
 
   /a/<job-name>/<job-id>/orig.config.yaml
 
-More details about config.yaml can be found on `detailed test config`_
+More details about config.yaml can be found at `detailed test config`_
 
 Triaging the cause of failure
 ------------------------------
 
-To triage a job failure, open the teuthology log for it using either(from the
-pulpito page):
+To triage a job failure, open the teuthology log for it using either the job
+name or the job id (from the pulpito page):
+
+   http://qa-proxy.ceph.com/<job-name>/teuthology.log
 
-http://qa-proxy.ceph.com/<job-name>/<job-id>/teuthology.log
+or
 
-and then opening log file with signature as:
+   http://qa-proxy.ceph.com/<job-id>/teuthology.log
```

Comment:
@zdover23 you don't need to change this, `http://qa-proxy.ceph.com/<job-name>/<job-id>/teuthology.log` is correct.

Reply:
I finally got around to fixing this.